src/operator/skip.ts

```diff
@@ -8,8 +8,8 @@ import { TeardownLogic } from '../Subscription';
  *
  * <img src="./img/skip.png" width="100%">
  *
- * @param {Number} the `n` of times, items emitted by source Observable should be skipped.
- * @return {Observable} an Observable that skips values emitted by the source Observable.
+ * @param {Number} n - The number of times, items emitted by source Observable should be skipped.
```

Comment:
The description reads: "Returns an Observable that skips `n` items emitted by an Observable."
If I change the param name to `total`, it will be confusing imo. Should I change the description as well?